cmd/graph/main.go

```diff
@@ -50,6 +51,11 @@ func main() {
 	mgr.Bootstrap()
 	mgr.SetIndexURLs(cfg.RepoDataURLs)
 
+	capacityProvider := scheduler.NewLocalCapacityProvider(appLogger, "build-packages")
```

Comment:
Hmm.  This can stay for now, but we should really come up with a better way to configure the scheduler.